packages/document-service-api/README.md

```diff
@@ -32,3 +32,19 @@ See `/src/lib/config.js`
 This application is built to the [12 Factor App](https://12factor.net/)
 standards. This means that there is a single configuration file and any specific
 variables required are declared as environment variables.
+
+### Accessing local blob storage.
+
+The document service has a local container that mimics that same functionality the azure blob storage provides, to browse the storage you need to download Azure Storage browser from [here](https://azure.microsoft.com/en-gb/features/storage-explorer/).
```

Comment:
The document service has a local container that mimics **the** same functionality **that** the **Azure** blob storage provides. **To** browse the storage you need to download Azure Storage browser from [here](https://azure.microsoft.com/en-gb/features/storage-explorer/).